### ISSUE TITLE
Build failing on Mac OS X with cmake

### ISSUE THREAD

**Issue by arkorobotics** [sfackler/cargo-tree#62]
I tried following the instructions by running `brew install cmake` and verified that I have version 3.14.3 installed.

After running `cargo install cargo-tree`, I get the following error:

```
error: failed to run custom build command for `miniz-sys v0.1.11`
process didn't exit successfully: `/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-64d3911af67afc86/build-script-build` (exit code: 101)
--- stdout
TARGET = Some("x86_64-apple-darwin")
OPT_LEVEL = Some("3")
HOST = Some("x86_64-apple-darwin")
CC_x86_64-apple-darwin = None
CC_x86_64_apple_darwin = None
HOST_CC = None
CC = None
CFLAGS_x86_64-apple-darwin = None
CFLAGS_x86_64_apple_darwin = None
HOST_CFLAGS = None
CFLAGS = None
CRATE_CC_NO_DEFAULTS = None
DEBUG = Some("false")
CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2,sse3,ssse3")
running: "cc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64" "-DMINIZ_NO_STDIO" "-DMINIZ_NO_ARCHIVE_APIS" "-DMINIZ_NO_ARCHIVE_WRITING_APIS" "-DMINIZ_NO_TIME" "-DMINIZ_NO_ZLIB_COMPATIBLE_NAMES" "-o" "/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/miniz.o" "-c" "miniz.c"
exit code: 0
AR_x86_64-apple-darwin = None
AR_x86_64_apple_darwin = None
HOST_AR = None
AR = None
running: "ar" "crs" "/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/libminiz.a" "/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/miniz.o"
cargo:warning=/opt/local/bin/ranlib: object: /var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/libminiz.a(miniz.o) malformed object (unknown load command 1)
cargo:warning=ar: internal ranlib command failed
exit code: 1

--- stderr
thread 'main' panicked at '

Internal error occurred: Command "ar" "crs" "/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/libminiz.a" "/var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/miniz.o" with args "ar" did not execute successfully (status code exit code: 1).

', /Users/arko/.cargo/registry/src/github.com-1ecc6299db9ec823/cc-1.0.35/src/lib.rs:2398:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

**Comment by sfackler** [comment]
This looks like a problem with your C toolchain:

```
cargo:warning=/opt/local/bin/ranlib: object: /var/folders/56/nrjnt_7502345jgyjl4c6lvh0000gn/T/cargo-installbg8DfW/release/build/miniz-sys-75966623faf4ac25/out/libminiz.a(miniz.o) malformed object (unknown load command 1)
cargo:warning=ar: internal ranlib command failed
```

**Comment by arkorobotics** [comment]
Not sure how I didn't see that. After some poking around, I discovered that I had both `ar` and `ranlib` installed in two different locations: `/opt/local/bin/` and `/usr/bin/`. My best guess is that I installed it in `/opt/local/bin` using homebrew some time in the past.

Regardless, I was able to resolve the issue my removing `ar` and `runlib` from `/opt/local/bin/`.

```
$ which ar
/opt/local/bin/ar
$ sudo mv /opt/local/bin/ar /opt/local/bin/ar-backup190427
$ sudo mv /opt/local/bin/ranlib /opt/local/bin/ranlib-backup190427
$ which ar
/usr/bin/ar
```

Builds just fine now, thanks!
